ext/session/tests/session_start_read_and_close.phpt

```diff
@@ -26,7 +26,7 @@ foreach ($valuesDisablingReadAndClose as $value) {
 
 try {
     session_start(["read_and_close" => 1.0]);
-} catch (Throwable $e) {
+} catch (TypeError $e) {
```

Comment:
Why this change?

Reply:
Because with the change I find it vague now we throw an explicit TypeError but ok with your other suggestions.